chainer/datasets/sub_dataset.py

```diff
@@ -115,6 +115,10 @@ def split_dataset_random(dataset, first_size):
     Args:
         dataset: Dataset to split.
         first_size (int): Size of the first subset.
+        seed (int): Seed the generator used for the permutation of indexes.
+            If an integer is specified, it is guaranteed that each sample
+            in the given dataset always belongs to a specific subset.
+            If ``None``, the permutation is changed randomly.
```

Comment:
It is better to note that `seed` must be 32 bit unsigned integers (or None), otherwise numpy will return `ValueError`.

See: https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.seed.html